### PR TITLE
feat: ticketing webshop form

### PR DESCRIPTION
**Pull request by jonasbrunvoll**
This PR provides a form for webshop, under the ticketing page.

### CODE REVIEW

File: src/translations/pages/contact.ts [hunk 1]
```diff
@@ -496,6 +496,21 @@ export const Contact = {
...
+        title: _(
+          'Hvilken billett gjelder det?',
+          'Which ticket is it about?',
+          'Kva billett gjeld det?',
+        ),
+        label: _('Kjøp billett', 'Purchase ticket', 'Kjøp billett'),
+      },
+      webshopAccount: {
+        label: _(
+          'Innlogging, profil, innstillinger',
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:509]: "Profil" eller "bruker"?

File: src/translations/pages/contact.ts [hunk 2]
```diff
@@ -951,6 +971,11 @@ export const Contact = {
 
     lastName: {
       label: _('Etternavn', 'Lastname', 'Etternamn'),
+      labelOptional: _(
+        'Etternavn (valgfritt)',
+        'Lastname (optional)',
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:976]:
```suggestion
        'Last name (optional)',
```

Eventuelt "Surname"?

File: src/translations/pages/contact.ts [hunk 3]
```diff
@@ -962,6 +987,12 @@ export const Contact = {
 
     email: {
       label: _('E-post', 'Email', 'E-post'),
+      labelOptionalIfCustomerNumberIsProvided: _(
+        'E-post (valgfritt hvis kundenummer er oppgitt)',
+        'Email (optional if customber number is provided)',
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:992]:
```suggestion
        'Email (optional if customer number is provided)',
```

File: src/translations/pages/contact.ts [hunk 4]
```diff
@@ -1018,6 +1049,11 @@ export const Contact = {
 
     phoneNumber: {
       label: _('Telefonnummer', 'Phonenumber', 'Telefonnummer'),
+      labelOptional: _(
+        'Telefonnummer (valgfritt)',
+        'Phonenumber (optinal)',
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:1054]:
```suggestion
        'Phone number (optional)',
```

File: src/translations/pages/contact.ts (hunk 1 above)
Comment (jonasbrunvoll) [reply src/translations/pages/contact.ts:509]: The specification from FRAM says `Innlogging, profil, innstillingar`

**Review by adriansberg — APPROVED** [review]

**Comment by adriansberg** [comment]
This form should not be available yet, right? Could only access it directly via the url.

**Comment by jonasbrunvoll** [comment]
> This form should not be available yet, right? Could only access it directly via the url.


Correct, none forms under `contact/ticketing` should be available yet in any other way than directly via the url.

**Pull request merged**